### ISSUE TITLE
Run after linting/prettier fixes have run?

### ISSUE THREAD

**Issue by wesbos** [typed-rocks/ts-worksheet#7]
Is there any way for it either re-run after fixes have taken place, or ignore things like prettier/eslint entirely? 

let me send a video of the issue im having:

https://share.cleanshot.com/xR6Msxpz

**Comment by chwoerz** [comment]
Really interesting. It would be nice if you could send me the code you are using. What I would suggest that you maybe could do is, that you run it using the command Palette. I've added the secion in the README. Just use `Run TypeScript Worksheet`. 
This works without the need to save. But I will look into it.

**Comment by wesbos** [comment]
Here is the code. Run typescript worksheet also give me the error:

```ts
// ts-worksheet-with-variables

const date = new Date();
console.log(date ); // ?

export {};

// const permissionsNeeded = new Set(['superadmin', 'admin', 'item:delete', 'item:update']);
// const myPermissions = new Set(['user', 'item:delete', 'item:read']);
// console.log(permissionsNeeded);

// const hi = 'hi'; // ?

// // What Permissions are they missing?
// const difference = permissionsNeeded.difference(myPermissions);
// // What Permissions do they both have?
// const intersection = permissionsNeeded.intersection(myPermissions);

// const inventory = new Set(['fries', 'ice cream']);
// const ordered = new Set(['fries', 'burger', 'shake']);

// // What are we missing?
// const missing = ordered.symmetricDifference(inventory);

// // Merge everything together
// const everything = ordered.union(inventory);
// // isDisjointFrom;
// // isSubsetOf;
// // isSupersetOf;
```

**Comment by chwoerz** [comment]
Super interesting. Thanks for providing me the code. It looks like that has something to do with eslint. Because for me it does not give any error when running the code. I will have to configure eslint. What kind of configuration you have? Would you mind sharing it?

**Comment by wesbos** [comment]
https://github.com/wesbos/hot-tips is the repo I'm working on, so you should be able to clone that and open in VS code.

I'm assuming its not compiling because VS code is showing errors in the document?

**Comment by chwoerz** [comment]
I will look into it. Thanks for your time. Really appreciate it.

**Comment by chwoerz** [comment]
Ah I see it. It really is the prettier issue. I will fix it.

**Comment by chwoerz** [comment]
@wesbos I released a new version 0.8.27 and it's fixed: 
<img width="532" alt="Bildschirmfoto 2024-10-15 um 23 23 14" src="https://github.com/user-attachments/assets/4f12669b-bba0-4604-9b2d-e4537cea8e3a">

Thanks for reporting it and giving it a try.

**Comment by wesbos** [comment]
you rule! thanks you. Both issues are fixed
